### PR TITLE
Copy custom CPU template JSON files to release directory

### PR DESCRIPTION
**Pull request by zulinx86**
## Changes

Add custom CPU template JSON files to release artifacts.

## Reason

We have custom CPU template JSON files under tests/data/static_cpu_templates that function equivalently as all the existing static CPUt templates. These JSON files are useful not only for us to check compatibility between static and custom ones, but also for users to migrate from static to custom ones smoothly.

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- ~~[ ] If a specific issue led to this PR, this PR closes the issue.~~
- [x] The description of changes is clear and encompassing.
- ~~[ ] Any required documentation changes (code and docs) are included in this PR.~~
- ~~[ ] API changes follow the [Runbook for Firecracker API changes][2].~~
- ~~[ ] User-facing changes are mentioned in `CHANGELOG.md`.~~
- [x] All added/changed functionality is tested.
- ~~[ ] New `TODO`s link to an issue.~~
- [x] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [x] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Comment by zulinx86** [comment]
Pushing this back to draft for further discussion.

File: tools/gh_release.py [hunk 1]
```diff
@@ -20,8 +20,19 @@
...
     # Do not include signatures in GitHub release since we aren't
-    # making those keys public
-    exclude_files = {"RELEASE_NOTES", "SHA256SUMS.sig"}
+    # making those keys public.
+    # Exclude CPU templates in GitHub release as they are already
+    # available on GitHub without any action (like building a binary).
+    exclude_files = {
+        "RELEASE_NOTES",
+        "SHA256SUMS.sig",
+        "c3.json",
```
Comment (pb8o) [inline tools/gh_release.py:29]: Could we do `*[x.stem for x in Path("tests/data/static_cpu_templates").glob("*.json")]`. That way we don't hardcode them.

**Review by pb8o — DISMISSED** [review]

File: tools/gh_release.py (hunk 1 above)
Comment (zulinx86) [reply tools/gh_release.py:29]: Looks better! I'll try to use it! Thanks.

Comment (zulinx86) [reply tools/gh_release.py]: applied your suggestion and confirmed it works! thanks!

**Review by pb8o — DISMISSED** [review]

**Comment by zulinx86** [comment]
needed to update this PR to remove a linking error.

**Review by kalyazin — APPROVED** [review]

**Review by pb8o — APPROVED** [review]

**Pull request merged**